jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/SchemaGenerator.java

```diff
@@ -75,4 +76,14 @@ public ObjectNode generateSchema(Type mainTargetType, Type... typeParameters) {
     public SchemaBuilder buildMultipleSchemaDefinitions() {
         return SchemaBuilder.forMultipleTypes(this.config, this.typeContext);
     }
+
+    /**
+     * Returns the {@link ObjectMapper} instance that is associated with the {@link SchemaGeneratorConfig} that was
+     * used to build this {@link SchemaGenerator}.
+     *
+     * @return {@link ObjectMapper}  instance
+     */
+    public ObjectMapper getObjectMapper() {
+        return this.config.getObjectMapper();
+    }
```

Comment:
**suggestion**: Let's add a `SchemaGeneratorConfig getConfig()` getter instead of this delegate.
The Maven plugin can then retrieve the `ObjectMapper` from that, without this being too specific here.